capstone/src/main/webapp/surveys/survey-download.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#Json file name where client secret is stored
+CLIENT_SECRET_FILE=''
+
+if [ -z "$CLIENT_SECRET_FILE" ]
+then
+  echo Store the name of the json file where the client secret is stored within the CLIENT_SECRET_FILE variable 
+  exit 1
+fi
+
+#download oauth21 tool
+wget https://storage.googleapis.com/oauth2l/latest/linux_amd64.tgz
+tar -xvf linux_amd64.tgz
+
+#id for each survey that is currently deployed
+SURVEY_IDS=('iilszvt24alys4gp7yyhypsuuq' 'edxr3yho5rkjsbcrepg7x4373e' 'wfj4mvsfw3mlcjvkoushioiomm' 'el6cjg2mb6rjbo5sel7lm3kkj4' 'rvhtbkditxingqxn5qoixau2eu'
```

Comment:
A future improvement might be to store these in a separate file instead of in the bash script.